tools/run_tests/xds_k8s_test_driver/framework/rpc/grpc.py

```diff
@@ -26,15 +26,26 @@
 
 
 class GrpcClientHelper:
-    channel: grpc.Channel
     DEFAULT_RPC_DEADLINE_SEC = 90
-
-    def __init__(self, channel: grpc.Channel, stub_class: Any):
+    channel: grpc.Channel
+    # This is purely cosmetic to make RPC logs look like method calls.
+    log_service_name: str
+    # This is purely cosmetic to output the RPC target. Normally set to the
+    # hostname:port of the remote service, but it doesn't have to be the
+    # real target. This is done so that when RPC are routed to the proxy
+    # or port forwarding, this still is set to a useful name.
+    log_target: str
+
+    def __init__(self,
+                 channel: grpc.Channel,
+                 stub_class: Any,
+                 *,
+                 log_target: Optional[str] = ''):
         self.channel = channel
         self.stub = stub_class(channel)
-        # This is purely cosmetic to make RPC logs look like method calls.
         self.log_service_name = re.sub('Stub$', '',
                                        self.stub.__class__.__name__)
+        self.log_target = log_target or ''
```

Comment:
Nit: Not sure what's going on here. Isn't the kwarg default already empty string?

Reply:
It is `Optional`, so technically someone could pass `None`. This is just a safeguard.